src/YoloDev.Expecto.TestSdk/main.fs

```diff
@@ -0,0 +1,4 @@
+module YoloDev.Expecto.TestSdk.Main
+
+[<EntryPoint>]
+let main _ = failwithf "YoloDev.Expecto.TestSdk is not intended to be run as a program"
```

Comment:
Is this file needed?

Reply:
If output type of the fsproj is exe then yes.